packs/feat-effects/effect-heavens-thunder.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Heaven's Thunder",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Heaven's Thunder]</p>\n<p>Your unarmed attacks and any monk weapons you wield deal additional electricity and sonic damage; the amount of each damage type is equal to the number of damage dice for the weapon or unarmed attack.</p>\n<p>If you also know the <em>ki blast</em> ki spell, you can choose to deal either electricity or sonic damage with the spell instead of force while this effect is active (<em>ki blast</em> loses the force trait and gains the electricity or sonic trait instead).</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Heaven's Thunder]</p>\n<p>Your unarmed attacks and any monk weapons you wield deal additional electricity and sonic damage; the amount of each damage type is equal to the number of damage dice for the weapon or unarmed attack.</p>\n<p>If you also know the <em>qi blast</em> ki spell, you can choose to deal either electricity or sonic damage with the spell instead of force while this effect is active (<em>qi blast</em> loses the force trait and gains the electricity or sonic trait instead).</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Heaven's Thunder]</p>\n<p>Your unarmed attacks and any monk weapons you wield deal additional electricity and sonic damage; the amount of each damage type is equal to the number of damage dice for the weapon or unarmed attack.</p>\n<p>If you also know the <em>qi blast</em> qi spell, you can choose to deal either electricity or sonic damage with the spell instead of force (<em>qi blast</em> loses the force trait and gains the electricity or sonic trait instead).</p>"
```